tests/ok_ko.ml

```diff
@@ -0,0 +1,27 @@
+(** Compile files in "OK" and "KO".
+    REVIEW: Most tests could be removed from "lp" files, and tested directly in
+    the code. *)
+open Core
+
+let _ = Files.set_lib_root (Some(".."));;
+
+let test_ok f () =
+  (* Simply assert that there is no exception raised. *)
+  Alcotest.(check unit) f (ignore (Compile.compile_file f)) ()
+
+let test_ko f () =
+  (* Succeed if compilation fails. *)
+  let r = try (ignore (Compile.compile_file f)); true with _ -> false in
+  Alcotest.(check bool) f r false
+
+let _ =
+  let open Alcotest in
+  let files = Sys.readdir "OK" |> Array.map (fun f -> "OK/" ^ f) in
+  (* TODO: replace the [f] of [test_case] by a keyword, possibly extracted
+     from the file.  *)
```

Comment:
What do you mean here?

Reply:
This one is not very important, but each test is tagged with a string, which is, for the moment, the filename.

Reply:
Why do you think it may be useful to use something else? Anyway, since this is not important, thanks to turn this "TODO" into a "REMARK".

Reply:
I've the todo it in my most recent commit, it's not important.